packages/fury-adapter-oas3-parser/lib/parser/oas/parseSchemaObject.js

```diff
@@ -58,10 +58,24 @@ function parseSchema(context) {
   const parseSubSchema = element => parseReference('schemas', R.uncurryN(2, parseSchema), context, element, true);
   const parseProperties = parseObject(context, `${name}' 'properties`, R.compose(parseSubSchema, getValue));
 
+  const arrayElementToParseResult = array => new namespace.elements.ParseResult(array.content);
+  const parseArray = itemParser => pipeParseResult(namespace,
```

Comment:
I wrote this in a way so we can extract it into a standalone component "parseArray" in the future